src/nxdoc/nuxeo-server/rest-api/rest-api-endpoints/batch-upload-endpoint.md

```diff
@@ -300,13 +300,13 @@ Before uploading any file, you need to initialize a batch, even if there is only
 This handshake phase is mandatory to acquire a server-side generated batch ID to be used in subsequent requests as part of the REST resource path.
 
 ```
-POST http://NUXEO_SERVER/nuxeo/api/v1/upload/
+POST http://NUXEO_SERVER/nuxeo/api/v1/upload/new/default
 ```
 
-This request returns a 201 CREATED status code with the following JSON data:
+This request initializes a new batch associated with the `default` handler and returns a 201 CREATED status code with the following JSON data:
 
 ```javascript
-{"batchId": batchId}
+{"provider": providerId, "fileEntries": [], "batchId": batchId}
```

Comment:
I think we should replace `providerId` by `default` as the example is on the `default` handler + add the same sentence & response in the _Using a Different Upload Handler_ paragraph by replacing `default` to `providerId` or `providerName` in the URL and the response.

Reply:
Agreed, sample response should be:
{"provider": "default", "fileEntries": [], "batchId": batchId}
Good suggestion on adding this sentence (with providerId) in the other section.